recipe/meta.yaml

```diff
@@ -16,10 +16,6 @@ build:
   # We skip the build on Windows because of linkage problems with libprotobuf.
   # See: https://github.com/man-group/ArcticDB/pull/449
   skip: true  # [win]
-  # The build of libprotobuf is lacking the _CFRelease symbol
-  # Fixing requires to rebuild the whole stack while OSX build
-  # for python 3.8 is not used  by anyone.
-  skip: true  # [osx and py<39]
```

Comment:
Build for python<3.9 aren't made anymore.

Hence this can and should be removed on other branches as well, removing the `conda-souschef` warning which was observed on this PR.